main.go

```diff
@@ -171,6 +150,36 @@ func main() {
 		os.Exit(1)
 	}
 
+	cm, err := util.GetCmOfMapCs(mgr.GetAPIReader())
```

Comment:
Given `bootstrap` is ready, we could replace `mgr.GetAPIReader()` by `bs.Reader`